setup.py

```diff
@@ -109,6 +109,7 @@ def get_setup_kwargs(raw=False):
             "Programming Language :: Python :: 3.9",
             "Programming Language :: Python :: 3.10",
             "Programming Language :: Python :: 3.11",
+            "Programming Language :: Python :: 3.12",
```

Comment:
If we add this then we should also update the version number in the readme :)

Reply:
the badge can't be updated to show 3.12, maybe it reads this setup.py change and updates it?

Reply:
The badge reads the python version from PyPi. What I mean is the version number in the introduction readme, but I just see that "3.8+" already covers that 🐳